versioned_docs/version-2.0.0/running-keploy/configuration-file.md

```diff
@@ -44,65 +56,198 @@ test:
+## Using the Config File
+
+The Keploy-config file eliminates the need to repeatedly specify parameters for each record or test command. The parameters in the file correspond to the flags in the Keploy [CLI Command Docs](http://keploy.io/docs/running-keploy/cli-commands/). To run a record or test command, use:
 
-To record, run:
+### Record Command:
 
 ```bash
 keploy record
 ```
 
-To test, run:
+### Test Command:
 
 ```bash
 keploy test
 ```
 
 Visit the [CLI Command Docs](http://keploy.io/docs/running-keploy/cli-commands/) to know more about the flags/parameters and their usage.
 
-The additional relief that the config file provides is that it allows the user to specify the `Global Noise` and `Test-Set Noise` in the config file itself. The global noise and test set noise are the [Noisy fields](http://keploy.io/docs/concepts/general-glossary/#3-noisy-field) that are needed to be ignore while comparing the response of the API calls. The global noise is the noise that is to be ignored for all the API calls, whereas the test-set noise is the noise that is to be ignored for a particular test set. The global noise and test-set noise are specified in the config file as shown below:
+## Configuration Sections
 
-```bash
-    globalNoise:
-      global:
-        body: {
-          # to ignore some values for a field,
-          # pass regex patterns to the corresponding array value
-          "url": ["https?://\S+", "http://\S+"],
-        }
-        header: {
-          # to ignore the entire field, pass an empty array
-          "Date: [],
-        }
-      # to ignore fields or the corresponding values for a specific test-set,
-      # pass the test-set-name as a key to the "test-sets" object and
-      # populate the corresponding "body" and "header" objects
-      test-sets:
-        test-set-1:
-          body: {
-            # ignore all the values for the "url" field
-            "uuid": ["b464d6df-d28b-4d12-8af1-0e2d61289578"]
-          }
-          header: {
-            # we can also pass the exact value to ignore for a field
-            "User-Agent": ["PostmanRuntime/7.34.0"]
-          }
+### Record Section
+
+The `record` section in the Keploy-config file allows you to define parameters for recording API calls.
+
+- **`path`**: Path to the project where recording occurs. (Mandatory field)
+
+- **`command`**: Command executed during recording.
+
+- **`proxyport`**: Port number for the proxy. Default is 0.
+
+- **`containerName`**: Name of the container during recording.
+
+- **`networkName`**: Network name for the container during recording.
+
+- **`delay`**: Delay in seconds before recording each request. Default is 5 seconds.
+
+- **`passThroughPorts`**: Ports passed through during recording.
+
```

Comment:
this field is removed now this works under stubs port.

Reply:
also you can explain some more about port, path and host in stubs and filters

Like if you see this 
```    # Example on using tests
    # tests: 
    #   filters:
    #    - path: "/user/app"
    #      urlMethods: ["GET"]
    #      headers: {
    #        "^asdf*": "^test"
    #      }
    #      host: "dc.services.visualstudio.com"
    # Example on using stubs
    # stubs: 
    #   filters:
    #    - path: "/user/app"
    #      port: 8080
    #    - port: 8081
    #    - host: "dc.services.visualstudio.com"
    #    - port: 8081
    #      host: "dc.services.visualstudio.com"
    #      path: "/user/app"
    #
    # Example on using globalNoise```
    
    user must be told how these fields can be used together as well as independently